Behavioral.Automation.Bindings.UI/Abstractions/IWebElement.cs

```diff
@@ -0,0 +1,12 @@
+using Behavioral.Automation.Bindings.UI.Context;
+
+namespace Behavioral.Automation.Bindings.UI.Abstractions;
+
+public interface IWebElement
+{
+    public WebContext WebContext { get; }
+    public ElementSelector ElementSelector  { get; }
+    public string? Description { get; set; }
+    public Task IsVisibleAsync();
+    public Task IsVisibleAsync(int delay);
```

Comment:
I meant `Ilocator` field from `IWebElement` interface :)

Reply:
I'll explain further: `ILocator` is a Microsoft-backed interface, it's a part of Playwright package, so it will most probably stay unchanged, so these visibility methods will just tangle things up

Reply:
But what if we will use these bindings with Selenium or Puppeteer. Do we need to have `ILocator` field in abstractions?

Reply:
@LukaRukhadze 
Changed back to ShouldBecomeVisibleAsync

Reply:
decided to get rid of asynchronous code

Reply:
I get the idea, of names of methods being the same. 
My main point was, that I don't like Method names written in Should form. If it is ok, with you guys, I'm ok with it.
@Oleg-Mozhey